services/bots/src/github-webhook/github-webhook.module.ts

```diff
@@ -28,7 +27,6 @@ import { ValidateCla } from './handlers/validate-cla';
   providers: [
     BranchLabels,
     CodeOwnersMention,
-    DependencyBump,
```

Comment:
We had this one because we don't trust users checking that box.

Since manifest.json is always ends up in `requirements_all.txt`, we could just drop `manifest.json` and `setup.py` to address the issue.